components/RepositoryList.tsx

```diff
@@ -18,10 +20,49 @@ const Loader = () => (
   </div>
 );
 
-export const RepositoryList = ({ repositories }: RepositoryListProps) => {
+export const RepositoryList = ({ repositories, dtOrder, starOrder  }: RepositoryListProps) => {
```

Comment:
Since the list can be ordered by date or by stars, I don't think we need both variables, but just one that tells us what we want to order by.